app/scripts/modules/core/src/search/searchResult/SearchResultGrid.tsx

```diff
@@ -57,7 +51,7 @@ export class SearchResultGrid extends React.Component<ISearchResultGridProps> {
     switch (status) {
       case SearchStatus.INITIAL:
       case undefined:
-        return <NoQuery/>;
```

Comment:
I can't for the life of me figure out where this ever would've gotten rendered, given that its parent component doesn't render unless there *is* a search query. Regardless, I got rid of it.

Reply:
this might have been a vestige of a previous version